common/djangoapps/student/views.py

```diff
@@ -402,13 +403,16 @@ def _cert_info(user, course_overview, cert_status, course_mode):  # pylint: disa
                 )
 
     if status in ('generating', 'ready', 'notpassing', 'restricted', 'auditing', 'unverified'):
```

Comment:
Nit: when checking for membership, use a set, for O(1) access time, instead of a tuple with O(N) access time.